### ISSUE TITLE
Restoring eyesync from backup does not ask for Contacts & Calendar permissions

### ISSUE THREAD

**Issue by joxlfg2q** [etesync/android#217]
### TL;DR
When etesync restored from backup it does NOT ask for Contacts & Calendar permissions.
As a result, no contacts & events are appearing despite sync working fine.

### Issue
New phone, restored backup from old phone using Neo Backup (APK + data), got prompted for Push Notifications permission.

Data was not transferred during the backup, my guess it's because of how it's stored on the device for security purposes.

Added account, sync was working fine, could see all the items, but no contacts nor events showing in the respective apps.

### Workaround
Reinstalling etesync made the permission dialogs show as expected.

### System information
Android 13
CalyxOS
Pixel 6a
etesync v2.3.0
